examples/themes/template-theme.html

```diff
@@ -0,0 +1,204 @@
+            <media-control-bar slot="top-chrome">
+              <template directive="if" expression="title">
+                <media-text-display part="top title display">
+                  <b>{{title | string}}</b>
+                </media-text-display>
+              </template>
+            </media-control-bar>
+            <div slot="centered-chrome" class="center-controls">
+              <!-- Here the modifier casts this expression value to a string, syntax is up to us. Can be changed -->
+              <media-play-button part="center play button" disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-play-button>
+            </div>
+            <media-control-bar>
+              <media-play-button disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-play-button>
+              <media-time-range disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-time-range>
+              <media-fullscreen-button disabled="{{disabledFullscreen}}" aria-disabled="{{disabledFullscreen | string}}">
+                <svg slot="enter" viewBox="0 0 24 24"><use xlink:href="#fullscreen-enter" /></svg>
+              </media-fullscreen-button>
+            </media-control-bar>
+          </template>
+
+          <!-- This could be a solution for when you'd like to re-use certain elements across different layouts -->
+          <!-- In this example the play and fullscreen button in the bottom control bar could be the same elements in extra small and large,
+                not working yet because they have to be cached in the template parts processor -->
+          <!-- It's quite verbose but this would be the foundation on which it is build, I could see we introduce non-HTML language
+                in the future but it would translate to what you see below at runtime if we chose template parts -->
+          <template directive="set" expression="centerPlayButton">
+            <media-play-button part="center play button" disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-play-button>
+          </template>
+          <template directive="set" expression="playButton">
+            <media-play-button disabled="{{disabled}}" aria-disabled="{{disabled | string}}"></media-play-button>
+          </template>
+          <template directive="set" expression="fullscreenButton">
+            <media-fullscreen-button disabled="{{disabled}}" aria-disabled="{{disabled | string}}">
+              <svg slot="enter" viewBox="0 0 24 24"><use xlink:href="#fullscreen-enter" /></svg>
+            </media-fullscreen-button>
+          </template>
+
+          <template directive="layout" expression="live extra-small">
+            <media-control-bar>
+              {{playButton}}
+              <div class="spacer"></div>
+              {{fullscreenButton}}
+            </media-control-bar>
+          </template>
+
+          <template directive="layout" expression="live large">
```

Comment:
Could this also work like a conditional? Like `directive="if" expression="?"`

Reply:
yes, there's one like that for title. 
https://github.com/muxinc/media-chrome/pull/362/files#diff-57ae49d38159fbdf0cae4bb887922912ee746d210c62a5f148f34998c55ace6fR72-R76

expression can contain all kinds of characters.
this is all possible if we'd like
https://github.com/dy/templize#expressions